frontend/summary/dataPivot/DataPivotExtension.js

```diff
@@ -144,80 +146,124 @@ _.extend(DataPivotExtension, {
+            _dpe_option_txt: "Show epi comparison set",
             hasModal: true,
         },
         {
             _dpe_name: "exposure",
             _dpe_key: "exposure id",
             _dpe_cls: Exposure,
-            _dpe_option_txt: "Show exposure",
+            _dpe_option_txt: "Show epi exposure",
             hasModal: true,
         },
         {
             _dpe_name: "outcome",
             _dpe_key: "outcome id",
             _dpe_cls: Outcome,
-            _dpe_option_txt: "Show outcome",
+            _dpe_option_txt: "Show epi outcome",
             hasModal: true,
         },
         {
             _dpe_name: "result",
             _dpe_key: "result id",
             _dpe_cls: Result,
-            _dpe_option_txt: "Show result",
+            _dpe_option_txt: "Show epi result",
             hasModal: true,
         },
         {
             _dpe_name: "meta_protocol",
             _dpe_key: "protocol id",
             _dpe_cls: MetaProtocol,
-            _dpe_option_txt: "Show protocol",
+            _dpe_option_txt: "Show epi meta protocol",
             hasModal: true,
         },
         {
             _dpe_name: "meta_result",
             _dpe_key: "meta result id",
             _dpe_cls: MetaResult,
-            _dpe_option_txt: "Show meta result",
+            _dpe_option_txt: "Show epi meta result",
             hasModal: true,
         },
         {
             _dpe_name: "iv_chemical",
             _dpe_key: "chemical id",
             _dpe_cls: IVChemical,
-            _dpe_option_txt: "Show chemical",
+            _dpe_option_txt: "Show invitro chemical",
             hasModal: true,
         },
         {
             _dpe_name: "iv_experiment",
             _dpe_key: "IVExperiment id",
             _dpe_cls: IVExperiment,
-            _dpe_option_txt: "Show experiment",
+            _dpe_option_txt: "Show invitro experiment",
             hasModal: true,
         },
         {
             _dpe_name: "iv_celltype",
             _dpe_key: "IVCellType id",
             _dpe_cls: IVCellType,
-            _dpe_option_txt: "Show cell type",
+            _dpe_option_txt: "Show invitro cell type",
             hasModal: true,
         },
         {
             _dpe_name: "iv_endpoint",
             _dpe_key: "IVEndpoint id",
             _dpe_cls: IVEndpoint,
-            _dpe_option_txt: "Show endpoint",
+            _dpe_option_txt: "Show invitro endpoint",
             hasModal: true,
         },
     ],
 });
 
+class ExtensionTable extends Component {
+    render() {
+        const columns = DataPivotExtension.extByColumnKey();
+        return (
+            <table className="table table-sm table-bordered">
+                <thead>
+                    <tr>
+                        <th>Input column header*</th>
+                        <th>Available action(s)</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {_.map(columns, (arr, col) => (
```

Comment:
I simplified this in two ways (hopefully you agree it's simpler):

1. used the lodash `_.map` instead of the built-in javascript, since we use lodash elsewhere it seems more readable
2. I treated interactive actions  as `ul` instead of tables cells; thus the logic on rowspans, etc wasn't needed

Reply:
Looks good!